examples/delegated-fungible/src/lib.rs

```diff
@@ -0,0 +1,115 @@
+// Copyright (c) Zefchain Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+/* ABI of the Delegated Fungible Token Example Application */
+
+use async_graphql::{scalar, Request, Response};
+pub use linera_sdk::{
+    abi::{ContractAbi, ServiceAbi},
+    abis::fungible::{Account, InitialState, Parameters},
+    graphql::GraphQLMutationRoot,
+    linera_base_types::{AccountOwner, Amount},
+};
+use serde::{Deserialize, Serialize};
+
+/// An ABI for applications that implement a fungible token.
+pub struct DelegatedFungibleTokenAbi;
+
+impl ContractAbi for DelegatedFungibleTokenAbi {
+    type Operation = Operation;
+    type Response = ();
+}
+
+impl ServiceAbi for DelegatedFungibleTokenAbi {
+    type Query = Request;
+    type QueryResponse = Response;
+}
+
+/// A delegated fungible operation
+#[derive(Debug, Deserialize, Serialize, GraphQLMutationRoot)]
+pub enum Operation {
+    /// Approve the transfer of tokens
+    Approve {
+        /// Owner to transfer from
+        owner: AccountOwner,
+        /// The spender account
+        spender: AccountOwner,
+        /// Maximum amount to be transferred
+        allowance: Amount,
+    },
+    /// Transfers tokens from a (locally owned) account to a (possibly remote) account.
+    Transfer {
+        /// Owner to transfer from
+        owner: AccountOwner,
```

Comment:
Maybe we should rename `AccountOwner` to address 🤔 I find this a bit confusing that `AccountOwner` is local account while `Account` is both `AccountOwner` and chain.

Reply:
I think the whole issue of renaming is for another discussion, PR and so on.

Reply:
I agree, I just wanted to share this observation when looking at these constructs in a new context.